docs/docs/dev_docs/getting_started/noir_contracts.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Setting up Aztec.nr Contracts
+---
+
+## Introduction
+
+This guide explains the necessary set up required to write your own contract using the Aztec.nr library and deploy it on the sandbox
+
+:::info Prerequisite reading
+If you haven't read [Aztec Sandbox](./sandbox.md) and [Noir](./noir.md), we recommend going there first.
+:::
+
+### Dependencies
+1. You will need the Noir build tool `nargo`, which you can install via [`noirup`](https://github.com/noir-lang/noirup). Make sure you install the `aztec` version of nargo:
+
+```bash
+curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
+noirup -v aztec
+```
+
+2. [Have a running sandbox and a code repo to interact with it](./sandbox.md)
```

Comment:
Would probably use a small header rather than the list. When looking over the steps it "vanishes in the crowd". Also would benefit by https://github.com/AztecProtocol/aztec-packages/pull/1770#discussion_r1304810188.